charts/merlin/templates/merlin-deployment.yaml

```diff
@@ -241,6 +241,12 @@ spec:
           {{- if .Values.authorization.enabled }}
           - name: AUTHORIZATION_SERVER_URL
             value: {{ include "merlin.authorization.server.url" . | quote }}
+          - name: AUTHORIZATION_CACHING_ENABLED
```

Comment:
@deadlycoconuts / @ariefrahmansyah, the new env var names are a little inconsistent. But, since work is underway to move away from env vars, to the configmap, I decided to not bother fixing this in the Merlin app.